scripts/west_commands/build.py

```diff
@@ -143,6 +144,10 @@ def do_add_parser(self, parser_adder):
                             dest='dry_run', action='store_true',
                             help='''Just print the build commands which would
                             have run, without actually running them.''')
+        parser.add_argument('-o', '--build-opt', default=[], action='append',
```

Comment:
`-o` is a bit weird but I really don't have a better suggestion

Reply:
Yeah, I struggled with this one.